chef-utils/lib/chef-utils/dsl/platform.rb

```diff
@@ -123,6 +123,21 @@ def centos_platform?(node = __getnode)
       # chef-sugar backcompat method
       alias_method :centos?, :centos_platform?
 
+      # Determine if the current node is CentOS Stream.
+      #
+      # @param [Chef::Node] node the node to check
+      # @since 17.0
+      #
+      # @return [Boolean]
+      #
+      def centos_stream_platform?(node = __getnode)
+        if node["lsb"].nil? || node["lsb"].empty?
```

Comment:
Shouldn't we just start with node['os_release'] and use lsb as a backup?

Reply:
That's what I'm doing here? So you'd rather I do `unless node.key?("os_release")` here?

Reply:
I guess it does the same thing, but it feels like the ask is backwards vs. just ``` if node["os_release"]```. It's really fine either way.

Reply:
I can change it to `if node["os_release"]` so that it makes more sense.